README.md

```diff
@@ -1,3 +1,21 @@
 # TextMining
 
-This is the base repo for the text mining and analysis project for Software Design at Olin College.
+## Description
+This project analyzes philosophical texts for linguistic similarity and visualizes their relationship spatially using Metric Multidimensional Scaling.
+It also includes a Markov text synthesizer to generate a philosophical "maxim" across all included schools of thought. 
+
+## Getting Started
+
+### Required Packages:
+pip install nltk requests vaderSentiment
+pip install matplotlib scikit-learn scip
```

Comment:
This is really minor, but these two lines came out as on the same line which made it initially difficult to understand that there were two commands. Make sure that your markdown styling is what you want it to be in the final product.